.github/workflows/BumpStdlibs.yml

```diff
@@ -2,18 +2,24 @@ name: BumpStdlibs
 on:
   workflow_dispatch:
     inputs:
+      BUMPSTDLIBS_TARGET_BRANCH:
+        description: 'Target branch (in the JuliaLang/julia) to target. Examples include master, release-1.9, etc.'
+        required: true
+        default: 'master'
```

Comment:
I think this will also need a source branch, given that both will be on `release-1.9` etc. ?

Reply:
But one could be on `backports-release-1.9` etc so can't use the same

Reply:
Hmmmm. Presumably both the source and target should be `backports-release-1.9`?

Reply:
I don't think so. backport- is internal staging for a repo, another shouldn't link to it, I believe

Reply:
I'm not sure I follow.

Reply:
We make our bump PR by (roughly speaking):

1. Clone our fork
2. In our fork, make sure that BRANCHNAME in our fork points to the same commit as BRANCHNAME in upstream.
3. Create a new branch off of BRANCHNAME in our fork.
4. In this new branch, perform the bump.
5. Commit and push our branch to our fork.
6. Open a PR from our fork to merge into BRANCHNAME in upstream.

It seems to me that BRANCHNAME should always have the same value, right?

It wouldn't make sense to branch off of `release-1.9` but then make a PR to merge that PR into `backports-release-1.9`, would it? What if the backports branch already has some modifications to the stdlib?

Reply:
backports-release-1.9 is only used on Pkg to collect backports, when its ready to be updated on julia it's merged into release-1.9, then in julia backports-release-1.9 will be updated to the tip of Pkg release-1.9. 

Backports branches aren't permanent things, just for PRs

Reply:
So for instance for a bump request, Pkg branch could be release-1.9 and julia branch be backports-release-1.9

They will often not match when managing release branches

Reply:
Oh, I see what you're getting at.

This branch name is only for the branch in upstream Julia.

The BumpStdlibs.jl package never takes in any input for the branch name in the stdlib repo. We always respect the stdlib branch specified as e.g. `PKG_BRANCH` in `Pkg.version`.

Reply:
E.g. we would parse https://github.com/JuliaLang/julia/blob/backports-release-1.8/stdlib/Pkg.version and then use the `release-1.8` branch of the Pkg.jl repo, because that's the value of `PKG_BRANCH`.

Reply:
So the user first needs to make a PR to e.g. edit the `PKG_BRANCH` variable in the `Pkg.version` file.

This can be done entirely from the GitHub web interface; if you only edit the `PKG_BRANCH` line (and you don't actually change the commit hash), you don't need to change any checksums.

Reply:
Ok. That seems reasonable